base/logging_context.go

```diff
@@ -22,3 +27,11 @@ func (lc *LogContext) addContext(format string) string {
 
 	return format
 }
+
+func FormatBlipContextID(contextID string) string {
+	return "[" + contextID + "]"
+}
+
+func FormatChangeCacheContextID(contextName string) string {
```

Comment:
Since we're generating a new ID here instead of just formatting the input, we should rename to reflect that.

```suggestion
func NewChangeCacheContextID(dbName string) string {
```